pkg/tfvars/openstack/bootstrap_ignition.go

```diff
@@ -137,6 +138,20 @@ func generateIgnitionShim(userCA string, clusterID string, bootstrapConfigURL st
 		},
 	}
 
+	var httpProxy, httpsProxy *string
+	if proxy.HTTPProxy != "" {
+		httpProxy = &proxy.HTTPProxy
+	}
+	if proxy.HTTPSProxy != "" {
+		httpsProxy = &proxy.HTTPSProxy
+	}
+
+	noProxySplitted := strings.Split(proxy.NoProxy, ",")
```

Comment:
```suggestion
	noProxySplit := strings.Split(proxy.NoProxy, ",")
```
variable grammer nit